lib/reporters/cli/index.js

```diff
@@ -81,9 +81,20 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
         // for some reason, if there is no run summary, it is unexpected and hence don't validate this
         var run = this.summary.run;
 
+        // show the body of the response for single request
+        // eslint-disable-next-line max-len
```

Comment:
I'm a little concerned that we're adding so many `eslint-disable` comments. Can we break this line properly and avoid these comments? 🤔